src/main/java/com/palantir/gradle/versions/ConsistentVersionsPlugin.java

```diff
@@ -31,10 +31,10 @@ public final void apply(Project project) {
         project.getPluginManager().apply(GetVersionPlugin.class);
 
         project.allprojects(proj -> {
-            proj.getPluginManager().withPlugin("java", plugin -> {
-                proj.getPluginManager().apply(FixLegacyJavaConfigurationsPlugin.class);
-            });
+            proj.getPluginManager()
+                    .withPlugin("java", plugin -> {
```

Comment:
Laying out this kind of chained method calls on a single line doesn't work yet but is fixable.